code/datums/wires/robot.dm

```diff
@@ -53,7 +53,6 @@
 			if(R.lawupdate)
 				R.visible_message("<span class='notice'>[R] gently chimes.</span>", "<span class='notice'>LawSync protocol engaged.</span>")
 				R.lawsync()
-				R.show_laws()
```

Comment:
Why was this removed?

Reply:
`lawsync` calls `post_lawchange` which calls `show_laws`,